Src/Plugins/Security/SecurePlaintextSecretsValidators/SEC101_006.GitHubLegacyPatValidator.cs

```diff
@@ -39,7 +39,7 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary
 
             // It is highly likely we do not have a key if we can't
             // find at least one letter and digit within the pattern.
-            if (!ContainsDigitAndChar(secret.Value))
+            if (!secret.Value.ToString().ContainsDigitAndLetter())
```

Comment:
>ToString() [](http://example.com/codeflow?start=30&length=10)

are these ToString() calls actually necessary?

I think secret.Value is already a string.

Reply:
I thought the same thing before. 
secret.Value is FlexString. 
secret.Value.ContainsDigitAndLetter() won't work here.

Reply:
<img width="1012" alt="image" src="https://user-images.githubusercontent.com/119978670/224439671-e88941fc-1b03-496b-b6d9-683f9f3175ed.png">